.zenodo.json

```diff
@@ -0,0 +1,130 @@
+{
+    "description": "The community-developed, free and open-source solar data analysis environment for Python.",
+    "license": "BSD-2-Clause",
+    "title": "SunPy",
+    "upload_type": "software",
+    "creators": [
+        {
+            "affiliation": "1 - DKIST. 2 - Aperio Software",
+            "name": "Stuart Mumford",
+            "orcid": "0000-0003-4217-4642"
+        },
+        {
+            "affiliation": "NASA Goddard Space Flight Center",
+            "name": "Steven Christe",
+            "orcid": "0000-0001-6127-795X"
+        },
+        {
+            "affiliation": "Aperio Software",
+            "name": "Nabil Freij"
+        },
+        {
+            "affiliation": "@google",
+            "name": "Florian Mayer"
+        },
+        {
+            "affiliation": "National Institutes of Health",
+            "name": "Keith Hughitt"
+        },
+        {
+            "affiliation": "1 - NASA Goddard Space Flight Center. 2 - Catholic University of America",
+            "name": "Daniel F. Ryan"
+        },
+        {
+            "name": "Simon Liedtke"
+        },
+        {
+            "affiliation": "NASA Goddard Space Flight Center",
+            "name": "Albert Y. Shih"
+        },
+        {
+            "affiliation": "University College London",
+            "name": "David Pérez-Suárez",
+            "orcid": "0000-0003-0784-6909"
+        },
+        {
+            "name": "Pritish Chakraborty"
+        },
+        {
+            "affiliation": "IIT Indore",
+            "name": "Vishnunarayan K I"
+        },
+        {
+            "name": "Andrew R. Inglis"
+        },
+        {
+            "affiliation": "IIIT Hyderabad",
+            "name": "Punyaslok Pattnaik"
+        },
+        {
+            "affiliation": "DIRAC Institute, UW",
+            "name": "Brigitta M. Sipocz",
+            "orcid": "0000-0002-3713-6337"
```

Comment:
@bsipocz is this correct?

Reply:
ORCID is correct, but how correct we should be with the affiliation? I wouldn't think UW is unambiguous enough.